ports/rp2/mpbthciport.c

```diff
@@ -50,46 +53,60 @@ void mp_bluetooth_hci_init(void) {
     events_task_is_scheduled = false;
 }
 
-STATIC void mp_bluetooth_hci_start_polling(void) {
-    events_task_is_scheduled = false;
-    mp_bluetooth_hci_poll_now();
-}
-
 static int64_t mp_bluetooth_hci_timer_callback(alarm_id_t id, void *user_data) {
     poll_timer_id = 0;
-    mp_bluetooth_hci_poll_now();
+    mp_bluetooth_hci_schedule_poll();
```

Comment:
I think it's better to keep the original name because it matches `stm32/mpbthciport.c`, and looks similar in name to the similar function `mp_bluetooth_hci_poll_in_ms()`.

If we wanted to rename it, maybe `mp_bluetooth_hci_poll_asap()` would be better.

Reply:
Fair enough. I guess I made the change because the rp2 port now only supports using the scheduler (unlike the stm32 port)